### PR TITLE
Install yarn and remove bower

### PR DESCRIPTION
**Pull request by victoria34**
Fix https://github.com/oncokb/oncokb/issues/1405

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,11 +2,65 @@
...
+    "@bower_components/levenshtein": "gf3/Levenshtein#~1.0.5",
+    "@bower_components/lodash": "lodash/lodash#~4.17.11",
+    "@bower_components/moment": "moment/moment#>=2.9.0",
+    "@bower_components/ng-tags-input": "mbenford/ngTagsInput-bower#3.2.0",
+    "@bower_components/oridomi": "dmotz/oriDomi#~1.1.5",
+    "@bower_components/please-wait": "Pathgather/please-wait#~0.0.2",
+    "@bower_components/spinkit": "tobiasahlin/SpinKit#~1.0.1",
+    "@bower_components/stringjs": "jprichardson/string.js#~2.2.0",
+    "@bower_components/uuid-js": "aurigadl/uuid-js#^0.7.5",
+    "@bower_components/x2js": "abdmob/x2js#~1.2.0"
```
Comment (zhx828) [inline package.json]: why are the components with bower prefix?

Comment (victoria34) [reply package.json]: I followed the official bower removal documentation. It introduces a bower-away package.
https://bower.io/blog/2017/how-to-migrate-away-from-bower
node_modules/@bower_components will contain all components in exactly the same way they would be installed by Bower

Comment (zhx828) [reply package.json]: @victoria34 should we use npm packages?

Comment (victoria34) [reply package.json]: I cannot find all corresponding bower packages in npm. If we really want to switch to NPM thoroughly, we may need to use some npm packages to replace bower components, fix version compatibility issue and rewrite code based on bower.

Comment (zhx828) [reply package.json]: @victoria34 it's not urgent, as long as the packages are under npm, I'm ok with that. Let's leave it like this for now.

Comment (victoria34) [reply package.json]: Thank you

**Pull request merged**